client/src/components/dialogs/AddPets.js

```diff
@@ -99,7 +43,7 @@ export default function AddPetsDialog(props) {
         profilePic: setProfilePic
     };
 
-    const classes = useStyles();
+    const classes = props.classes;
```

Comment:
Use the spread operator to get classes here (also why not use hooks for styling and not HOC)